man/2/demi_init.md

```diff
@@ -0,0 +1,59 @@
+# `demi_init(2)`
+
+## Name
+
+`demi_init` - Initializes Demikernel.
+
+## Synopsis
+
+```c
+#include <demi/libos.h>
+
+int demi_init(int argc, char *const argv[]);
+```
+
+## Description
+
+`demi_init()` initializes Demikernel. It sets up devices, instantiates LibOSes, and performs general initialization tasks.
+
+The `argv` parameter is an array of argument strings passed to Demikernel. The `argc` parameter is a positive integer that specifies the length of that array.
+
+The array of argument strings provides Demikernel with various information that are critical for its initialization. For instance, these arguments shall state which LibOSes should be initialized, and whether or not runtime features shall be turned on.
+
+All arguments supported by Demikernel are listed in the table bellow. Arguments that are not listed therein are ignored.
+
+|ID       | Argument      | Description             |
+|---------|---------------|-------------------------|
+| `ARG-1` | `--catcollar` | Enables Catcollar LibOS |
+| `ARG-2` | `--catnap`    | Enables Catnap LibOS    |
+| `ARG-3` | `--catnip`    | Enables Catnip LibOS    |
+| `ARG-4` | `--catpowder` | Enables Catpowder LibOS |
+
+ Any constraints/restrictions of arguments are detailed next:
+
+- `ARG-1` takes effect only on Linux hosts. If this is the only parameter passed to Demikernel, `demi_init()` will fail.
```

Comment:
What other parameter needs to be passed along with this one (--catcollar) for demi_init() not to fail?  Below it says these four (LibOSes) are mutually exclusive, so presumably none of those, and no other possible arguments/parameters are listed.